corporate/lib/decorator.py

```diff
@@ -43,19 +43,19 @@ def _wrapped_view_func(
 
 def authenticated_remote_realm_management_endpoint(
     view_func: Callable[Concatenate[HttpRequest, RemoteRealmBillingSession, ParamT], HttpResponse]
-) -> Callable[Concatenate[HttpRequest, ParamT], HttpResponse]:  # nocoverage
+) -> Callable[Concatenate[HttpRequest, ParamT], HttpResponse]:
     @wraps(view_func)
     def _wrapped_view_func(
         request: HttpRequest,
         /,
         *args: ParamT.args,
         **kwargs: ParamT.kwargs,
     ) -> HttpResponse:
-        if not is_self_hosting_management_subdomain(request):
+        if not is_self_hosting_management_subdomain(request):  # nocoverage
             return render(request, "404.html", status=404)
 
         realm_uuid = kwargs.get("realm_uuid")
-        if realm_uuid is not None and not isinstance(realm_uuid, str):
+        if realm_uuid is not None and not isinstance(realm_uuid, str):  # nocoverage
             raise TypeError("realm_uuid must be a string or None")
```

Comment:
I think this is impossible; given how these endpoints are declared in `urls.py`, I think we could just write it as `*, realm_uuid: str` (i.e. list it as a mandatory kwarg) rather than args/kwargs in the parameters, and delete this check.

Similarly for the `RemoteZulipServer` case below.